ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -97,7 +96,7 @@ cat > packet-setup.yaml <<-EOF
         fi
 
         IP=\$(cat "\${SHARED_DIR}/server-ip")
-        SSHOPTS=(-o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' -i "\${CLUSTER_PROFILE_DIR}/.packet-kni-ssh-privatekey")
+        SSHOPTS=(-o 'ConnectTimeout=5' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=90' -i "\${CLUSTER_PROFILE_DIR}/ssh-key")
```

Comment:
If I'm not wrong there's already _another_ `ssh-key` (coming from cluster profile), I think this could generate some clashing, maybe it's better to keep the packet one with a different name

Reply:
How about `packet-ssh-key`?